pmm-tests/pg_stat_monitor_setup.sh

```diff
@@ -59,43 +75,33 @@ export PATH="/usr/lib/postgresql/${pgsql_version}/bin:$PATH"
 echo $PATH
 cp /usr/lib/postgresql/${pgsql_version}/bin/pg_config /usr/bin
 
-## Down PGSM repo and move to /home/postgres/pg_stat_monitor dir
-##
-if [ -z "$pgstat_monitor_branch" ]
-then
-      export pgstat_monitor_branch=REL_1_STABLE
-fi
-
-## Down PGSM repo and move to /home/postgres/pg_stat_monitor dir
-##
-if [ -z "$pgstat_monitor_repo" ]
-then
-      export pgstat_monitor_repo=percona/pg_stat_monitor
-fi
-
+# Clone PGSM repo and move to /home/postgres/pg_stat_monitor dir
 cd /home/postgres
-git clone -b ${pgstat_monitor_branch} https://github.com/${pgstat_monitor_repo}
+git clone -b ${pgstat_monitor_branch} https://github.com/percona/pg_stat_monitor
```

Comment:
any specific reason for removing this? We needed this when people are using their forked repo and trying to run the builds, so qa-integration and github action calls with their custom repos/branches

Reply:
Sure, I will revert it back.